### PR TITLE
Make a big mental shift

### PR DESCRIPTION
**Pull request by whit537**
This is it, the mental shift we [need to make](https://github.com/gratipay/inside.gratipay.com/issues/1160#issuecomment-332998643) to get out of our own heads and into our customer's.

"Pay for open source" is about _our_ problems and motivations.

"Invest in open source" is about _their_ problems and motivations.

### CODE REVIEW

File: deploy/before.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    ALTER TABLE payments_for_open_source RENAME TO investments_in_open_source;
```
Comment (rohitpaulk) [inline deploy/before.sql]: @whit537 I understand that you're experimenting here, but aren't there quicker/cheaper/safer ways to achieve this than having to rename this throughout the codebase? 😕

Comment (rohitpaulk) [reply deploy/before.sql]: My suggestion would be to only tinker with copy. Modeling investments as `payments` in code is forgivable, IMO - not worth spending your time changing

Comment (whit537) [reply deploy/before.sql]: Fair enough. I've pushed 49c96e72efd71de6a362846cf695dd73f6cba5ba off, we can come back to it later if @JessaWitzel wants to. :D

Comment (mattbk) [reply deploy/before.sql]: Agree with @rohitpaulk.

**Comment by whit537** [comment]
@JessaWitzel Merge? 😄

**Comment by whit537** [comment]
Okay stopping. [slack](https://gratipay.slackarchive.io/gratipay/page-100/ts-1506793372000009) 🐭

**Pull request merged**